pkgs/by-name/ae/aegis-rs/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  stdenv,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "aegis-rs";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "Granddave";
+    repo = "aegis-rs";
+    rev = "v${version}";
+    hash = "sha256-QqOZybggHD2+W51+4r1rp2YA4oyJnXsuXdYJCbkOpHo=";
+  };
+
+  cargoHash = "sha256-9V4bBgEeuhz008Su4Z1WEVY4CpIbk8BuyRD2vIxtcWY=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    openssl
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    darwin.apple_sdk.frameworks.Security
+    darwin.apple_sdk.frameworks.SystemConfiguration
```

I think `apple_sdk` has been migrated to `apple-sdk` and no longer has these attributes.
```suggestion
    darwin.apple-sdk
```

---

you shouldn't need to add `darwin.apple-sdk` either, its in the darwin stdenv